actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -88,7 +88,12 @@ def translate(key, options = {})
           raise e if raise_error
 
           keys = I18n.normalize_keys(e.locale, e.key, e.options[:scope])
-          content_tag('span', keys.last.to_s.titleize, :class => 'translation_missing', :title => "translation missing: #{keys.join('.')}")
+          title = "translation missing: #{keys.join('.')}"
+          interpolation_options = options.except(:default)
+          if interpolation_options.present?
+            title << ", #{options.map{|k,v| "#{k}: #{ERB::Util.html_escape(v.to_s)}"}.join(', ')}"
```

Comment:
Also `ERB::Util.html_escape` calls `to_s` for us. So need for that.

Lastly some style. Add spaces and the `map` braces: `map { |k,v| "long interpolation" }.join(', ')` :wink: